firstvoices/backend/views/import_job_views.py

```diff
@@ -44,7 +53,47 @@
         ],
     ),
     create=extend_schema(
-        description=_("Add a batch import job."),
+        description=_(
+            "Queue a new batch import job and create a model instance to track the job. The first stage "
+            "of the import job workflow is to validate and execute a dry-run of the provided CSV, and "
+            "display any issues with the CSV or rows. It also displays how many rows will be imported,"
+            " ignored or contain any errors with their details. Once the dry-run has been successfully "
+            "completed, the confirm action can be used to do the database import for the provided CSV."
```

Comment:
+1 for adding these details. I think we could tweak this to focus more on how to use the API instead of describing what the code does. Maybe something like:

'Creates a new batch import job and automatically starts generating a validation report. Once the validationStatus is "COMPLETE", the validationReport will list how many rows can be imported and any errors or ignored columns or rows. See the "confirm" API to import the data.'